src/components/Timer/timer.test.ts

```diff
@@ -0,0 +1,42 @@
+import { Challenge } from '../../model/ChallengeConfiguration'
+import track from '../../examples/ian-motorcycle-ride-path'
+import { getTimes } from './timer'
+import { DateTime } from 'luxon'
+/**
+ * import gpx path data
+ * read data from file and get string from data
+ * convert string data to be a TrackPath
+ */
+const testTimer = test('returns time from component', () => {
+  const defaultChallenge: Challenge = {
+    start: {
```

Comment:
I spotted something that doesn't look quite right. 

Compare the "start" object to the "finish" object. They have the same values!

A start line and finish line should almost always have different values, otherwise the time will be zero!

You can fix this manually by looking up some real coordinates on google maps and using them instead. Be sure they intersect the roads you use or you will get "undefined" for one of the lines.